probes/hasBadgeNotFound/def.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasBadgeNotFound
+short: This check determines whether the project is missing an OpenSSF (formerly CII) Best Practices Badge.
+motivation: >
+  The OpenSSF Best Practices badge indicates whether or not that the project uses a set of security-focused best development practices for open source software.
```

Comment:
nit: remove `that`

Reply:
I think this sentence is misleading. It's possible to follow the best practices without having that badge. I get that the point is to push this thing everywhere for whatever reason but it would be nice if scorecard could make it clear that the absence of the badge doesn't mean what this sentence says it means.

Reply:
Removed as suggested in https://github.com/ossf/scorecard/pull/3520#discussion_r1338975296